### PR TITLE
feat(AnimixPlay): add new presence

### PR DESCRIPTION
**Pull request by SafuGit**
## Description 
Add Presence for AnimixPlay.

## Acknowledgements
- [ ] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [ ] I linted the code by running `yarn format`
- [ ] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image
image
image

</details>

### CODE REVIEW

**Comment by theusaf** [comment]
Note: previously removed in #7084

**Review by Bas950 — DISMISSED** [review]

File: websites/A/AnimixPlay/metadata.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+		"en": "Animixplay offers the best user experience unmatched by any other platform. Little to no ads, a friendly and ergonomic user interface, an incredibly well organised watchlist, and easy access to all the information you could possibly want about any anime."
+	},
+	"url": [
+		"animixplay.fun",
+		"animixplay.best",
+		"www1.animixplayer.top",
+        "animixplay.st"
+	],
+	"version": "1.0.0",
+	"logo": "https://animixplay.fun/themes/zanimix/icon.png",
```
Comment (theusaf) [inline websites/A/AnimixPlay/metadata.json:18]: Logo should be 512x512px

File: websites/A/AnimixPlay/presence.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
+const presence = new Presence({
+		clientId: "1278380348731818081",
+	}),
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	strings = presence.getStrings({
+		play: "presence.playback.playing",
+		pause: "presence.playback.paused",
+	}),
```
Comment (theusaf) [inline websites/A/AnimixPlay/presence.ts:8]:
```suggestion
```

File: websites/A/AnimixPlay/presence.ts [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: Assets.Logo,
+			buttons: [
+				{
+					label: "Visit AnimixPlay",
+					url: "https://animixplay.fun/",
+				},
+			],
```
Comment (theusaf) [inline websites/A/AnimixPlay/presence.ts:32]: Buttons like these are prohibited based on the guidelines.
```suggestion
```

File: websites/A/AnimixPlay/presence.ts [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+	)
+		presenceData.details = "Browsing Anime";
+	presenceData.startTimestamp = browsingTimestamp;
+
+	if (href.startsWith("https://www1.animixplayer.top/")) {
+		const [startTimestamp, endTimestamp] = presence.getTimestamps(
+				Math.floor(video.currentTime),
+				Math.floor(video.duration)
+			),
+			TitleAndEpisode = document.querySelector(".animetitle").textContent;
```
Comment (theusaf) [inline websites/A/AnimixPlay/presence.ts:48]: variable name should be in `camelCase`

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/A/AnimixPlay/presence.ts [hunk 5]
```diff
@@ -0,0 +1,66 @@
+const presence = new Presence({
+		clientId: "1278380348731818081",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets { // Other default assets can be found at index.d.ts
+	Logo = "https://animixplay.fun/themes/zanimix/icon.png",
```
Comment (theusaf) [inline websites/A/AnimixPlay/presence.ts:7]:
```suggestion
	Logo = "https://i.postimg.cc/9XLKNDt8/icon-animixplay.png",
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by theusaf — APPROVED** [review]

**Pull request merged**